xpdan/data_reduction.py

```diff
@@ -80,7 +80,7 @@ def _feature_gen(self, event):
         for key in self.data_fields:
             val = event['data'].get(key, None)
             if el is not None:
-                feature = "{}={}".format(key, val)
+                feature = "{}".format(val)
```

Comment:
We need to add a temperature or some other variable into the `insert_imgs` function.